alces-flight-tutorials/metadata.yml

```diff
@@ -0,0 +1,75 @@
+---
+install:
+  el6: |
+    yum install -y -e0 nodejs npm yarn
+  el7: |
+    yum install -y -e0 nodejs npm yarn
+  ubuntu1604: |
+    apt-get install -y -e0 nodejs npm yarn
+  _: |
+    require serviceware
+    if serviceware_is_installed alces-flight-www; then
+        cp etc/alces-flight-www/tutorial.conf.tpl \
+          "${cw_ROOT}"/etc/alces-flight-www/server-https.d/tutorial.conf
+        # XXX Does this work with relative paths?
```

Comment:
I don't remember, but I suspect not.  You might get some mileage out of:

`$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)` to get the absolute path.